### PR TITLE
Update the README to look a little more pleasing

### PR DESCRIPTION
**Pull request by Krutonium**
Specifically I added the Logo to the top and centered it, as well as doing a minor re-wording of the first sentance, making it one sentance with a comma insted of 2 separate ones.

The Logo also links to OpenRCT2.io

https://github.com/OpenRCT2/OpenRCT2/blob/9e9709fc1a50d9d3325a788542cf4e272e81de3a/readme.md
If you want a quick preview.

### CODE REVIEW

**Review by tupaschoal — APPROVED** [review]
Here's a new (top) vs old (bottom) comparison for the sake of documenting:
image

It looks great to me

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
-# OpenRCT2
-An open-source re-implementation of RollerCoaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.
+
+
+<p align="center">
```
Comment (AaronVanGeffen) [inline readme.md:3]: `align="center"` is deprecated HTML. Use `style="text-align: center"` instead. (Same below)

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+  <a href="https://openrct2.io">
+    <img src="https://raw.githubusercontent.com/OpenRCT2/OpenRCT2/develop/resources/logo/icon_x128.png" width="128px" alt="OpenRCT2 Logo"/>
```
Comment (AaronVanGeffen) [inline readme.md:5]: `width="128px"` is invalid HTML. Either use `width="128"` (deprecated) or `style="width: 128px"`

**Comment by AaronVanGeffen** [comment]
Not a big fan of adding HTML to an otherwise Markdown-only document. But if we do go down that road, let it be modern HTML.

**Comment by Gymnasiast** [comment]
Annoyingly, that does not seem to work. It gets left-aligned :/

**Comment by Broxzier** [comment]
GitHub is a bit strict with what styles it allows to be used. To center text horizontally, you'll have to resort to `<p align="center">`, unfortunately.

**Comment by Gymnasiast** [comment]
Ugh, taking a leaf out of Outlook’s HTML handling book...

It’s unfortunate, as I do agree it looks a bit better.

@OpenRCT2/developers what do you think, should be pull this with icky HTML or reject this?

**Comment by tupaschoal** [comment]
I'm not very HTML-y myself, so I can't argue on the technicality. If it looks better and is very self-contained, I'm ok, but if it seems troublesome, then I'd drop it.

**Comment by Broxzier** [comment]
Since it's only used for GitHub, I'm fine with it getting added to an otherwise HTML-only document. It's still readable and simple to understand.

**Review by Broxzier — APPROVED** [review]

**Comment by Gymnasiast** [comment]
Do not merge yet, my commit should be dropped.

**Comment by Broxzier** [comment]
@Gymnasiast Sorry I clicked 'accept' too soon, I was going to squash and rebase it. I'm done now.

**Comment by Gymnasiast** [comment]
Why though? GitHub handles the squash fine.

**Comment by Broxzier** [comment]
To get rid of the extra commits here, but this works too.

**Pull request merged**